pkg/otbuiltin/builtin.go

```diff
@@ -112,8 +112,8 @@ func generateError(err *C.GError) error {
 	return goErr
 }
 
-// isOk wraps a return value (gboolean/gint) into a bool.
+// isOk wraps a return value gboolean into a bool.
```

Comment:
Minor nit: "a gboolean return value".

Reply:
thanks for the quick review, I've fixed it now